ivadomed/loader/slice_filter.py

```diff
@@ -40,24 +47,24 @@ def __init__(self, filter_empty_mask=True,
             else:
                 self.classifier = torch.load(classifier_path, map_location='cpu')
 
-    def __call__(self, sample):
+    def __call__(self, sample: dict):
         input_data, gt_data = sample['input'], sample['gt']
```

Comment:
Minor line shift change. I think this line went above comment.

Reply:
Good catch! Resolved in b111bfb.